Database/Corrections/sodQuestFixes.lua

```diff
@@ -135,6 +135,10 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.preQuestSingle] = {78092},
             [questKeys.zoneOrSort] = sortKeys.PALADIN,
         },
+        [78114] = {
+            [questKeys.questLevel] = -1,
```

Comment:
I think the questLevel should stay at level 25, since you have to fight some high level mobs. `-1` means that the questLevel will always match the level of the player and therefore will never show as "red" or "green".

Reply:
I've done it, and it requires that you get a random drop from level 23-25 mobs in Ashenvale, and a guaranteed drop from the end boss in Wailing Caverns which is a level 22 elite mob. Should we instead have it set to level 22?

Reply:
Hmm... The `questLevel` is like a "suggested level the player should have". So I think 22 is fine as well, yeah.